source/usage-examples/command.txt

```diff
@@ -0,0 +1,46 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. A raw database operation is a
+command you can execute directly on the MongoDB server CLI. Commands
+include administrative and diagnostic tasks, such as fetching server
+stats or initializing a replica set. Call the ``runCommand()`` method
+with a Bson command object on an instance of a database to run your raw
```

Comment:
Nitpick: should've picked this up before, but since we're specifying types, how about ``MongoDatabase`` instead of database to make it clear for a first-time MongoDB user?